src/sharedframework/framework/project.json

```diff
@@ -5,7 +5,8 @@
     },
 
     "dependencies": {
-        "Microsoft.NETCore.App": "1.0.0-rc2-23911"
+        "Microsoft.NETCore.App": "1.0.0-rc2-23911",
+        "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160318-02"
```

Comment:
What do you mean? We needed a fix from the newest build of this package which removed a problematic dependency. I'm not familiar with how Roslyn is versioning their packages, but there is no newer package with version 1.2.* than about a month ago.